Backpack/FlareView/Classes/BPKFlarePath.h

```diff
@@ -51,7 +53,9 @@ IB_DESIGNABLE @interface BPKFlarePath : NSObject
  *
  * @param size CGSize of the path required.
  */
-+ (UIBezierPath *)flareViewPathForSize:(CGSize)size NS_SWIFT_NAME(makeFlarePath(size:));
++ (UIBezierPath *)flareViewPathForSize:(CGSize)size
```

Comment:
Is this exposed externally?

Reply:
It's in the `.h` file so I guess?

Reply:
So then this is breaking?

Reply:
Oh sorry my mistake - this is not imported into `FlareView.h`, so it's only publicly available if someone imports `Backpack/BPKFlarePath.h` which is unsupported